sdk/src/beta9/runner/function.py

```diff
@@ -131,87 +125,120 @@ def main(channel: Channel):
     gateway_stub: GatewayServiceStub = GatewayServiceStub(channel)
     task_id = config.task_id
 
-    with StdoutJsonInterceptor(task_id=task_id), ThreadPoolExecutorOverride() as thread_pool:
+    if not task_id:
+        raise RunnerException("Invalid runner environment")
+
+    start_time = time.time()
```

Comment:
Why did  we need to move the start_time out?

Reply:
Just moved a lot around. Its moved back to where it was.